src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -167,7 +163,11 @@ public String getApiUri() {
             for (GHRepository repo : user.listRepositories()) {
                 add(listener, observer, repo);
             }
+            return;
         }
+
+        listener.getLogger().format("%n%s does not correspond to a known GitHub User Account or Organization%n%n", repoOwner);
+        throw new InterruptedException();
```

Comment:
@KostyaSha `InterruptedException` does not involve an error following the documentation.